rfcs/README.md

```diff
@@ -0,0 +1,75 @@
+
+# Roadmap to Open Source Release
```

Comment:
nit: probably rephrase because we're already 'released as open source', this is more about when we tag one and try to promote it in the ecosystem.